src/sentry/integrations/vsts/issues.py

```diff
@@ -37,7 +37,7 @@ def get_project_choices(
         try:
             projects = client.get_projects(self.instance)
         except (ApiError, ApiUnauthorized, KeyError) as e:
-            self.raise_error(e)
+            raise self.raise_error(e)
```

Comment:
I think your PRs are getting mixed up? This is from https://github.com/getsentry/sentry/pull/30577/files